From a9252d845480c6d2181c69271e2a1c2467d6d8f6 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Wed, 9 Dec 2015 18:34:13 +0000
Subject: [PATCH] Accomodate MSys2's libgc API

The libgc shipped with MSys2 declares the `GC_set_warn_proc()` function
as returning void, so let's not use its return value.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.c b/main.c
index b421943..e318cbc 100644
--- a/main.c
+++ b/main.c
@@ -833,7 +833,7 @@ main(int argc, char **argv, char **envp)
     mySignal(SIGPIPE, SigPipe);
 #endif
 
-    orig_GC_warn_proc = GC_set_warn_proc(wrap_GC_warn_proc);
+    /* orig_GC_warn_proc = */ GC_set_warn_proc(wrap_GC_warn_proc);
     err_msg = Strnew();
     if (load_argc == 0) {
 	/* no URL specified */
-- 
2.6.2

